docs/tutorials/optimize/memory/memap.md

```diff
@@ -1,10 +1,10 @@
-### memap - static memory map analysis
+## memap - static memory map analysis
```

Comment:
I don't want to change heading levels until after we move pages. Please fix all heading levels to how they were before.